### ISSUE TITLE
Add option to provide separate config file [$25]

### ISSUE THREAD

**Issue by wuuti** [modmore/Gitify#204]
### Summary
Currently the .config-file is always used. It would be great to have an option
```
 --config <file>
```
where another config file can be provided. 

This is useful e.g. for local dev installations where you have multiple "deployment" tasks in your wrapper scripts.

**Comment by ashu-22** [comment]
What Kind of steps Should I follow up on this request?

**Comment by hugopeek** [comment]
@ashu-22 You mean to claim the bounty? You'll need to create a pull request with the addition that @wuuti described above. Once that gets accepted and merged into the Gitify repository, the bounty is yours :)

Basically, you need to add the ability to run the following commands:
```
gitify build --config <file>
gitify extract --config <file>
```
Where <file> is a valid Gitify configuration file, other than the main .gitify.

**Comment by ryu0** [comment]
Has this been resolved yet?

**Comment by Bruno17** [comment]
I've solved multiple configs by creating a folderstruture like that:
modxroot/
-gitify/
--configA/
--configB/

and have a config.core.php within the config - directories with 
```
<?php

include (dirname(dirname(dirname(__FILE__))) . '/' . 'config.core.php' );
```
running Gitify inside the config - folders

**Comment by hugopeek** [comment]
@Bruno17 Nice and elegant alternative! Works for me :+1:

**Comment by Bruno17** [comment]
@Mark-H  maybe worth to put it to the docs?

**Comment by wuuti** [comment]
So that has been merged #417
